deploy/docs/Best_Practices.md

```diff
@@ -201,7 +201,7 @@ You can find more information on the `grep` filter plugin in the [fluentd docume
 If you want to capture container logs to a container that writes locally, you will need to ensure the logs get mounted to the host so fluent-bit can be configured to capture from the host.
 
 Example:
-In the fluentbit overrides file (https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/fluent-bit/overrides.yaml) in the `rawConfig section`, you have to add a new input specifying the file path, eg.
+In `values.yaml` in the `fluent-bit.rawConfig` section, you have to add a new input specifying the file path, eg.
```

Comment:
This will need modification once we upgrade the fluent-bit helm chart, as raw config goes away